packages/interfaces/src/protocols.ts

```diff
@@ -26,9 +27,20 @@ export type ContentTopicInfo = {
   contentTopics: string[];
 };
 
-export type ShardingParams = ShardInfo | ContentTopicInfo;
+export type ApplicationInfo = {
+  clusterId: number;
+  application: string;
+  version: string;
+};
+
+export type ShardingParams = ShardInfo | ContentTopicInfo | ApplicationInfo;
 
 export type ProtocolCreateOptions = {
+  /**
+   * @deprecated
+   * Waku will stop supporting named sharding. Only static sharding and autosharding will be supported moving forward.
+   */
+  pubsubTopics?: PubsubTopic[];
```

Comment:
@adklempner can you, please, create a task with description of what should be removed to completely deprecate parameters/functionality? So that we address it in a skip release or something

Reply:
I opened https://github.com/waku-org/js-waku/issues/1775